base/linalg/dense.jl

```diff
@@ -5,6 +5,7 @@
 ## BLAS cutoff threshold constants
 
 const SCAL_CUTOFF = 2048
+const TRIA_CUTOFF = 2048
```

Comment:
You are right, but it seems `sizeof` applied on the matrix works: `sizeof(Integer[0x1, 0x2]) == 16` (which in my understanding is the size of the pointers, so the cutoff would still probably apply, even if the elements themselves (pointed at) could be scattered in memory, so performance difference could be less visible).